server/http/restapi/asset.go

```diff
@@ -0,0 +1,27 @@
+package restapi
+
+import (
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+)
+
+func (a *restapi) handleGetStaticAsset(w http.ResponseWriter, req *http.Request, actingUserID string) {
+	query := req.URL.Query()
+
+	assetName := query.Get("name")
+	appID := query.Get("app_id")
```

Comment:
1/5 Using query strings to indicate which asset is requested seems a bit verbose to type. Maybe we want to use the URL path to indicate that i.e. `/static/$APP_ID/$NAME`